core/tauri/scripts/core.js

```diff
@@ -132,28 +132,113 @@
   }
 
   // drag region
-  document.addEventListener('mousedown', (e) => {
-    if (e.target.hasAttribute('data-tauri-drag-region') && e.buttons === 1) {
+  /**
+   * @param {boolean} isClick
+   * @param {MouseEvent} event
+   */
+  function handleDrag(isClick, event) {
+    /**
+     * @typedef {Object} DragInfo
+     * @property {boolean} drag
+     * @property {boolean} container
+     * @property {boolean} title
+     * @property {boolean} interactable
+     */
+
+    /**
+     * @param {HTMLElement} element
+     * @return {DragInfo|null}
+     */
+    function elementDragInfo(element) {
+      const dragAttr = element.getAttribute("data-tauri-drag-region");
+      const containerAttr = element.getAttribute("data-tauri-drag-region-container");
+      const titleAttr = element.getAttribute("data-tauri-drag-region-title");
+      const interactableAttr = element.getAttribute("data-tauri-drag-region-interact-able");
```

Comment:
no need for extra dash

```suggestion
      const interactableAttr = element.getAttribute("data-tauri-drag-region-interactable");
```